Formula/flex.rb

```diff
@@ -41,6 +41,12 @@ def install
       system "./autogen.sh"
     end
 
+    # Fix segmentation fault during install on Ubuntu 18.04 (caused by glibc 2.26+), which
+    # remove with the next release
```

Comment:
Whoops, broke the grammar
```suggestion
    # Fix segmentation fault during install on Ubuntu 18.04 (caused by glibc 2.26+),
    # remove with the next release
```